assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.js

```diff
@@ -115,7 +115,7 @@ export default function SettingsEdit() {
 	}
 
 	return (
-		<div className="googlesitekit-setup-module googlesitekit-setup-module--reader-revenue-manager">
+		<div className="googlesitekit-setup-module googlesitekit-setup-module-edit googlesitekit-setup-module--reader-revenue-manager">
```

Comment:
This additional class name should not be needed as we should be able to determine the screen based on already available selectors.